node-tests/helpers/expect-coffee.js

```diff
@@ -0,0 +1,10 @@
+var coffeescript = require('coffee-script');
+var expect = require('ember-cli-blueprint-test-helpers/chai').expect;
+
+module.exports = function(file) {
+  var compileFunc = function() {
+    coffeescript.compile(file.content);
+  }
+
+  expect(compileFunc).to.not.throw(Error);
```

Comment:
no problem. Just tried it out with a random string but apparently thats valid coffeescript so it doesn't throw. However calling `coffeescript.compile('var x = 1;');` does throw an error. Do you want me to add a unit test in for the helper?